Logstash/logstash-input-azurewadtable/lib/logstash/inputs/azurewadtable.rb

```diff
@@ -52,14 +53,34 @@ def run(output_queue)
   def teardown
   end  
 
-  def process(output_queue)
-    @logger.debug(@last_timestamp)
+  def build_latent_query
+    @logger.debug(@until_timestamp)
```

Comment:
to improve log readability, could you please log something like  "from _latest_timestamp_ to _until_timestamp_" and remove _lastest_timestamp_ logging in _process_ method. Or alternatively log "from - to" in the _proccess_ method